docs/topics/migrations.txt

```diff
@@ -467,6 +467,74 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. _data_migrations_and_multiple_databases:
+
+Data migrations and multiple databases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One of the challenges that can be encountered is figuring out whether or
+not to run a migration when using multiple databases.
+
+For example, you may want to **only** run a migration against a particular
+database.
+
+In order to do that you can filter inside of your data migration
+by looking at the ``schema_editor.connection.alias`` attribute.
+However it is better to have your ``dbrouter`` leverage the same
+``allow_migrate`` call by making the following method in your
+router::
+
+        class MyRouter(object):
+
+            def allow_migrate(self, db, model):
+                # your typical migration code here
+
+            def runpython_ok(self, apps, schema_editor, model):
```

Comment:
This function seems like code duplication. Can't we directly leverage the `allow_python` in `RunPython`?